lib/tests/unittests/HttpFileTest.cc

```diff
@@ -0,0 +1,52 @@
+#include "../../lib/src/HttpFileImpl.h"
```

Comment:
Can we include the MultiPart.h instead of the impl class to avoid the symbol issue of the shared lib building on windows?